src/CommunityToolkit.Maui.Core/Interfaces/IDockLayout.shared.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace CommunityToolkit.Maui.Core.Interfaces;
+
+/// <summary>Docking position for a view in <see cref="IDockLayout"/>.</summary>
+public enum DockEnum
```

Comment:
Let's rename this enum to `DockPosition`
```suggestion
public enum DockPosition
```